cmd/pipeline/main.go

```diff
@@ -84,10 +84,11 @@ import (
 	"github.com/banzaicloud/pipeline/internal/cluster/clusterdriver"
 	"github.com/banzaicloud/pipeline/internal/cluster/clustersecret"
 	"github.com/banzaicloud/pipeline/internal/cluster/clustersecret/clustersecretadapter"
+	"github.com/banzaicloud/pipeline/internal/cluster/distribution/awscommon/awscommonadapter"
```

Comment:
I don't really like this package structure. `awscommon` is not a distribution. Please try to come up with some alternatives that we can choose from.

For example: common AWS code can be grouped under an `infrastructure/aws` package. (I won't accept this package structure unless you come up with at least two other alternatives)